### ISSUE TITLE
Close main window with CMD+W

### ISSUE THREAD

**Issue by krstp** [zed-industries/community#1360]
### Check for existing issues

- [X] Completed

### Describe the feature

After closing all the TAB there is no option to close open empty window. Good example is when I have multiple windows open with multiple tabs. Sometimes I just want to close one of those... so I hit CMD+W to close the tabs, but when no tabs are open this particular window would still stay open. It would be also nice if there was a prompt at the end asking user if user really wants to close the main window.

**Comment by wzxu** [comment]
Hmm when I read this title, I thought you meant for Zed to allow us to close empty window with cmd-W instead of cmd-shift-W, which is how VS Code does it as well. I'd love this.
<img width="633" alt="image" src="https://user-images.githubusercontent.com/5619348/231201900-8810c49b-4e06-465d-92bc-256265f8f7a7.png">
<img width="653" alt="image" src="https://user-images.githubusercontent.com/5619348/231202077-a1468b94-ec1c-4c88-abea-3e21973ff833.png">

But after reading the description it's pretty different from I imagined. Now I'm not sure if I should open a new issue for what I described above… 🤔

**Comment by krstp** [comment]
Yes, CMD+W; this is what I had in mind.

**Comment by porsager** [comment]
On my end VSCode also closes the window with CMD+W if there are no tabs left. This is how any MacOS app behaves as far as I know. Would be great if Zed also did it :)

**Comment by fuzenco** [comment]
Yes, please. Command+W should close any open window if no tabs are open. In fact…

I would love if Zed closed the last remaining window automatically if there’s only one tab open across the app. I frequently just work with one file at a time and having window remain open (with the last closed tab’s title still remaining in the window title area) is disconcerting to me.
